class FailureException {
  final dynamic message;

  FailureException(this.message);
}